protobuf/flow/executiondata/executiondata.proto

```diff
@@ -25,4 +34,34 @@ message GetExecutionDataByBlockIDRequest {
 
 message GetExecutionDataByBlockIDResponse {
   entities.BlockExecutionData block_execution_data = 1;
+}
+
+message SubscribeExecutionDataRequest {
+  bytes start_block_id = 1;
+  uint64 start_block_height = 2;
+}
+
+message SubscribeExecutionDataResponse {
+  uint64 block_height = 1;
+  entities.BlockExecutionData block_execution_data = 2;
+  google.protobuf.Timestamp block_timestamp = 3;
+}
+
+message SubscribeEventsRequest {
+  bytes start_block_id = 1;
+  uint64 start_block_height = 2;
+  EventFilter filter = 3;
+}
+
+message SubscribeEventsResponse {
+  bytes block_id = 1;
+  uint64 block_height = 2;
+  repeated entities.Event events = 3;
+  google.protobuf.Timestamp block_timestamp = 4;
+}
+
+message EventFilter {
+  repeated string event_type = 1;
+  repeated string address = 2;
+  repeated string contract = 3;
```

Comment:
For people with a lack of context you may give a couple of examples of filters to highlight that they are OR'ed.